Makefile

```diff
@@ -44,6 +44,14 @@ APPS2 := kmesh-cmd
 APPS3 := mdacore
 APPS4 := kmesh-cni
 
+TMP_FILES := bpf/kmesh/bpf2go/bpf2go.go \
+	bpf/kmesh/bpf2go/kmeshcgroupsockworkload_bpfeb.go \
```

Comment:
The two .go files are intermediate compilation files and should not be archived. Suggest adding to .gitignore

Reply:
? can not ignore, they are committed

Reply:
Do you mean we should remove them?

Reply:
yes, previously mistakenly added.